aws-java-sdk-core/src/main/java/com/amazonaws/retry/PredefinedBackoffStrategies.java

```diff
@@ -147,4 +144,10 @@ public final long delayBeforeNextRetry(AmazonWebServiceRequest originalRequest,
         }
     }
 
-}
\ No newline at end of file
+    private static final long exponentialDelayHelper(int retriesAttempted, int baseDelay, int maxBackoffTime) {
```

Comment:
Looks like that as a part of running "mvn" locally the  pom.xml file gets changed.
I will rebase again against master and get rid of this change.

I am fine with the name suggestion.